test/e2e/scenario/rum/init.scenario.ts

```diff
@@ -108,7 +108,7 @@ describe('API calls and events around init', () => {
 describe('beforeSend', () => {
   createTest('allows to edit non-view events context')
     .withRum({
-      beforeSend(event) {
+      beforeSend: (event: any) => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I believe you've got some "implicit `any`" error because you didn't build packages with `yarn build` before running typechecks.

This was already the case before your PR, this is why [we build before typechecking e2e tests](https://github.com/DataDog/browser-sdk/blob/3ab528b0711a721d17b2c50e757fed5ece92f4c0/.gitlab-ci.yml#L134-L137). I think you can revert your modification in `init.scenario.ts` `logs.scenario.ts`.

Another solution could be to always import from `.../src/...` instead of relying on built packages, but that's a bigger change and a bit out of scope.

Reply:
Indeed - The implicit `any` was due to a TS server de-sync. 
However - I need to keep the new form: `beforeSend: (event) => {}` For some reason, after the updates to `tsconfig.json` (I suspect target="ES2015"), the test failed with the former code.